binding/Binding/Skottie/Animation.cs

```diff
@@ -0,0 +1,91 @@
+﻿using System;
+using System.ComponentModel;
+using System.IO;
+using System.Runtime.InteropServices;
+
+namespace SkiaSharp.Skottie
+{
+	public unsafe class Animation : SKObject
```

Comment:
Similar to the invalidation, we can add `ISKSkipObjectRegistration` to prevent the locking of the handle dictionary and storage of the object if the instance is never returned by an API. It is via Create, but that is more of a constructor. If the object cannot be obtained at a later stage, then it is free to skip registration.